tutor/src/screens/assignment-review/grant-extension.js

```diff
@@ -180,6 +180,7 @@ const ExtendModal = observer(({ ux, form: { isValid, values } }) => {
                             disabledDate={ux.course.isInvalidAssignmentDate}
                             timezone={ux.course.timezone}
                             validate={(d) => { // eslint-disable-line consistent-return
+                                d = new Time(d)
                                 if (!d) { return 'must be valid date' }
```

Comment:
since `d` is now a Time class you can use `d.isValid` here

Reply:
Good idea, swapped to that. Hard to test though because the picker automatically overrides it? 😆